keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/CloudDriverCache.kt

```diff
@@ -0,0 +1,13 @@
+package com.netflix.spinnaker.keel.clouddriver
+
+import com.netflix.spinnaker.keel.clouddriver.model.Network
+import com.netflix.spinnaker.keel.clouddriver.model.SecurityGroup
+
+interface CloudDriverCache {
```

Comment:
The idea is this class is a higher level layer around CloudDriver that we can apply cacheing to and thus freely use it to do the kind of id -> name transforms or "what are all the AZs in region x" type queries.